en/04_Changelogs/5.2.0.md

```diff
@@ -157,6 +157,12 @@ Both of these fields include a `setIsLazyLoaded()` method which will load a limi
 
 Note that these are both powered by react components, and are only intended to be used within the CMS. If you want to use them on the front-end of your project you will need to provide your own templates and JavaScript implementation for them.
 
+#### Auto scaffolding of `has_one` relations
+
+A `SearchableDropdownField` will now be used when automatically scaffolding `has_one` relations on a form in the CMS. Previously a `DropdownField` was used, and when there were over 100 items a [`NumericField`](api:SilverStripe\Forms\NumericField) was used which was not user friendly.
```

Comment:
```suggestion
A `SearchableDropdownField` will now be used when automatically scaffolding `has_one` relations on a form in the CMS. Previously a `DropdownField` was used, and when there were over 100 items a [`NumericField`](api:SilverStripe\Forms\NumericField) was used which was not user friendly.
```

Please add a note here about the change to `dropdown_field_threshold` (i.e. it controls the threshold for lazy loading now)

Reply:
Done